devops/arangodb.yml

```diff
@@ -9,6 +9,12 @@
     - name: arangodb_root_password
       prompt: "Password for ArangoDB root account"
 
+    - name: arango_readonly_username
+      prompt: "Username for ArangoDB readonly user"
+
```

Comment:
Mentioned elsewhere but we should be able to cut this bit out.

Reply:
16ac1d5